Sources/StrongContractClient.Request/FrontBackModels/FrontBackModels/URL+Calls/URLQueryItem.swift

```diff
@@ -6,10 +6,11 @@
 //  Copyright © 2018 ElevatedUnderdogs. All rights reserved.
 //
 
-import CoreLocation
-import os.log
-
+import Foundation
+#if canImport(os)
+import os
 let log = OSLog(subsystem: "com.ElevatedUnderdogs.akin1", category: "networking")
+#endif
```

Comment:
I think I did a log handling for different availabilities somewhere... I'll have to find that.